libs/ktem/ktem/db/base_models.py

```diff
@@ -95,3 +96,78 @@ class BaseIssueReport(SQLModel):
     chat: Optional[dict] = Field(default=None, sa_column=Column(JSON))
     settings: Optional[dict] = Field(default=None, sa_column=Column(JSON))
     user: Optional[int] = Field(default=None)
+
+
+class TagType(str, Enum):
+    text = "Text"
+    classification = "Classification"
+    boolean = "True/False"
```

Comment:
as for Enum, let's keep the enum value simple like `TEXT`, `CLASSIFICATION`, `TRUEFALSE`, all lower/uppercased without symbols